Dockerfile.dapper

```diff
@@ -14,7 +14,7 @@ ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm=armv6l GOLANG_ARCH=GOLANG_ARCH_${ARC
     GOPATH=/go PATH=/go/bin:/usr/local/go/bin:${PATH} SHELL=/bin/bash
 
 RUN wget -O - https://storage.googleapis.com/golang/go1.11.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local && \
-    go get github.com/rancher/trash && go get golang.org/x/lint/golint
+    go get github.com/rancher/trash && curl -L https://git.io/vp6lP | sh
```

Comment:
can you switch the short URL to `https://raw.githubusercontent.com/alecthomas/gometalinter/v2.0.12/scripts/install.sh`